_posts/2017-07-29-css-grids.md

```diff
@@ -0,0 +1,170 @@
+---
+layout: post
+author: Cyan/Jos
+title: Grid CSS
+---
+
+### What is CSS Grids?
+CSS Grids is a new standard in CSS3 and is slowly being adopted by all web browsers.
+Creating a basic website like this one requires a few lines of CSS, for example:
+
+![Example Site]({{ site.url }}/assets/css-grids/example-site.png)
+
+And the CSS:
+
+```css
+.wrap {
+    border: 5px solid red;
+    height: 400px;
+}
+
+.main-content {
+    width: 45%;
+    margin-left: 1%;
+    height: 98%;
+    float: left;
+    background-color: aqua;
+}
+
+.sidebar {
+    width: 45%;
+    margin-right: 1%;
+    height: 98%;
+    float: right;
+    background-color: crimson;
+}
+```
+
+---
+
+Grid creates, well, a grid inside a div element (Think of lines between the space in the image below). 
+
+![Grid Layout]({{ site.url }}/assets/css-grids/grid-layout.png)
+
+---
+
+We can decide how many rows and columns we create as well as the gaps between them.
+After this any child elements (we cant place any child of child elements, that would be too inception-y) can be formed to fit inside this grid, which eliminates thinking about how to design a site using dozens of wrappers and placing elements inside boxes inside boxes.
+
+```css
+.wrap {
+    border: 5px solid red;
+    height: 400px;
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: 100%;
+    grid-gap: 10px;
+}
+
+.main-content {
+    margin-left: 1%;
+    grid-area: 0/1;
+    height: 98%;
+    background-color: aqua;
+}
+
+.sidebar {
+    margin-right: 1%;
+    grid-area: 1/2;
+    height: 98%;
+    background-color: crimson;
+}
+
+```
+
+This does the same thing, what we are doing is making a grid with 2 columns and one row, then setting the columns size to `1fr` which is 1 fraction of the available space
```

Comment:
> This does the same thing,

This should be one sentence.

>This does the same thing. What we are doing is..

Also, missing a period at the end of this sentence.